src/TockContext.tsx

```diff
@@ -7,15 +7,29 @@ import React, {
   Context,
   useContext,
 } from 'react';
+import deepmerge from 'deepmerge';
+import { PartialDeep } from 'type-fest';
 import { retrieveUserId } from './utils';
 
+export const TockConfigContext: Context<TockConfig | undefined> = createContext<
```

Comment:
I would avoid abbreviation Config -> Configuration

Reply:
What about `Setting` ? Less verbose, isn't it?

Reply:
`TockSettings` sounds good